### PR TITLE
Allow password from env vars

### PR DESCRIPTION
**Pull request by sevastos**
Be able to retrieve the password from environment variables

e.x.:
```sh
PASSWORD=itsasecret storjfarm start
# or 
export PASSWORD=itsasecret
storjfarm start
```

If you find the naming of `PASSWORD` too generic, let me know.
Cheers

### CODE REVIEW

**Comment by bookchin** [comment]
Let's use `STORJSHARE_PASSPHRASE` for the environment variable name.

File: bin/storjshare.js [hunk 1]
```diff
@@ -112,8 +112,9 @@ var ACTIONS = {
       }
     }
 
-    if (env.password) {
-      open(env.password, privkey);
+    var password = env.password || process.env.PASSWORD;
```
Comment (bookchin) [inline bin/storjshare.js]: For style consistency, can you add another line break here?

**Comment by bookchin** [comment]
Also, mind adding a note to the README?

**Comment by sevastos** [comment]
Sure thing, updated, let me know if there's anything more.

**Comment by bookchin** [comment]
Sweet, looks good! Will include in the next release.

**Pull request merged**